### ISSUE TITLE
basic particles

### ISSUE THREAD

**Issue by Bayesianworld** [aapm-bdsc-ontology-tg/radont#35]
Andre,
Looks like you have imported some CHEBI classes.  We had done this before and had them all located under material entity -> subatomic particle (also a chebi class).  Looks like subatomic particle class has been removed and all the child classes moved up a level and some classes, like boson removed.  Is this a result of doing the chebi import.  Personally I liked the chebi class structure better the way before--it better reflects my thinking of these things.

**Comment by andredekker** [comment]
Yes, imported CHEBI incl. some more exotic ions that CHEBI defined for us. My reasoning to put everything under molecular entity was that 
1. Subatomic particles do not include ions, so this was not a superclass for all ionizing particles anyway. (note that Chebi includes protons as a subatomic particle although a hydrogen atom is a proton so in that case it is not subatomic).
2. I think we should include only things that are relevant for our domain. In our domain there is such a thing like 
a. hadron therapy (although many people would not include neutrons in that?) and 
b. ion therapy (although i think many people would not include alpha particles in that and also may not include protons - so this warrants discussion / custom definition in an application ontology). 
That is why I included hadrons and ions but not subatomic particles or the boson concepts.
3. Another thing i was wondering if we need to define subclasses of photons which describe how they were created (gamma, x-ray, bremmstrahlung) -> will we use that?

Let's discuss this at the next meeting as well?
